versions/v1.0.0/Social/Profile/Update.adoc

```diff
@@ -20,6 +20,19 @@ interface Update {
 }
 ----
 
+===== Fields
+
+`action` *Required*:: Action of updating a profile, e.g. `update`.
+`profile_id` *Optional*:: Updated ID of the profile, e.g. `0x01766e`.
+`address` *Optional*:: Updated address of the profile, e.g. `0x76d4ffda8c66864fa07f0a285b8e01e11b4bb7b3`.
+`handle` *Optional*:: Updated handle/username of the profile, e.g. `brinse.lens`.
+`image_uri` *Optional*:: Updated image URI of the profile, e.g. `https://media.orb.ac/thumbnailDimension768/https://statics-polygon-lens.s3.eu-west-1.amazonaws.com/profile/nft-0x6441543C6537FF430f2e1d6fE8FFFcfc05d52D04_polygon_0x5D666F215a85B87Cb042D59662A7ecd2C8Cc44e6_3097622.gif`.
+`bio` *Optional*:: Updated biography of the profile.
```

Comment:
what does biography mean in this context?

Reply:
The brief introduction of a person? Perhaps just using a bio is sufficient.